### ISSUE TITLE
Unable to run a packaged python program with standard socket libary

### ISSUE THREAD

**Issue by Domii894** [pyinstaller/pyinstaller#1501]
Hello pyinstaller team!

I just tried to pack a little py file to a mac-executable, but that didn't work out correctly.

I tried to pack a py file with following code in it:
```python
import socket
print("hi")
input()
```

and tried to pack it into an executable with following command:
```
pyinstaller -F test.py
```

I am working currently on a mac (Yosemite 10.10.5 (14F27)) with the newest python version (3.5) and the newest pyinstaller version (from GitHub (python3 branch) - last commit: https://github.com/pyinstaller/pyinstaller/commit/f716cbf86806de0b47cf1f4eb96cc958b7e4c77f)

The packaging seemed to run successful, but after I tried to run the file in the "dist"-folder, following error occured:

```
MACBOOKNAME:~ MYNAME$ /path/to/file ; exit;
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 664, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 634, in _load_backward_compatible
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/PyInstaller-3.0.dev3-py3.5.egg/PyInstaller/loader/pyimod03_importers.py", line 311, in load_module
    exec(bytecode, module.__dict__)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/socket.py", line 49, in <module>
    import _socket
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 664, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 634, in _load_backward_compatible
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/PyInstaller-3.0.dev3-py3.5.egg/PyInstaller/loader/pyimod03_importers.py", line 492, in load_module
    loader = _frozen_importlib.ExtensionFileLoader(fullname, filename)
AttributeError: module '_frozen_importlib' has no attribute 'ExtensionFileLoader'
test returned -1
logout
```

I tried to pack some other modules, which worked fine for me. The program executed and I was able to use it as expected. Only the *socket* module (and socket dependent modules like ftplib, ...) do not work.

I think it's an import issue from pyinstaller but I am not quite sure. So I hoped to find some help here :)

I hope, that this error report helps you out a bit. If you need some more information, feel free to ask me here.

Thanks for helping me.

**Comment by htgoebel** [comment]
Python 3.5 is not supported, see https://github.com/pyinstaller/pyinstaller/wiki

**Comment by Domii894** [comment]
Oh, sorry. I didn't see that :/
Thanks for the quick response.
Do you know, when python3.5 will be supported?

**Comment by codewarrior0** [comment]
It looks like the frozen importlib is slightly different from the normal one on 3.5.

The ExtensionFileLoader class is moved to `_frozen_importlib._bootstrap_external.ExtensionFileLoader`

**Comment by matysek** [comment]
@Domii894 could you try again? This commit might fix this issue: https://github.com/pyinstaller/pyinstaller/commit/f54579ca3a76c78f89f8f26985f1e6ccb80c6818

**Comment by Domii894** [comment]
Yeah I will do that
I've installed python 3.4 now and won't switch back to python 3.5 that early.
The issue is closed and python 3.5 is still not supported, but I promise, that I will.

**Comment by matysek** [comment]
current status of python 3.5 support is experimental. We made some fixes for Python 3.5.
